tests/unittests.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+import os
+import sys
+import unittest
+from pathlib import Path
+
+import PIL
+
+FILE = Path(__file__).resolve()
+ROOT = FILE.parents[1]
+if str(ROOT) not in sys.path:
+    sys.path.append(str(ROOT))
+ROOT = Path(os.path.relpath(ROOT, Path.cwd()))
+
+from src.pyimg_converter import convert_from_path
```

Comment:
Nitpick: PEP 8: E402 module level import not at top of file
You can use `isort` to automate this.